src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonic.hpp

```diff
@@ -333,41 +342,60 @@ struct EvolutionMetavars {
       Initialization::Actions::AddComputeTags<
           tmpl::list<evolution::Tags::AnalyticCompute<
               volume_dim, analytic_solution_tag, analytic_solution_fields>>>,
-      GeneralizedHarmonic::Actions::InitializeDampedHarmonicRollonGauge<
```

Comment:
How does this PR interact with #2237?

Reply:
I'm not sure, the PR you linked doesn't change any actions. If changes to these actions are coming up that interact with the initialization procedure perhaps we should talk briefly in a hangout